src/annotator/config/settings.js

```diff
@@ -1,10 +1,10 @@
-import * as sharedSettings from '../../shared/settings';
+import { parseJsonConfig } from '../../boot/parse-json-config';
 
 import configFuncSettingsFrom from './config-func-settings-from';
 import isBrowserExtension from './is-browser-extension';
 
 export default function settingsFrom(window_) {
-  const jsonConfigs = sharedSettings.jsonConfigsFrom(window_.document);
+  const jsonConfigs = parseJsonConfig(window_.document);
```

Comment:
As we make tests more modern and consistent, I wonder if we can knock it off with these test seams and just mock the needful things in the tests...but maybe there's another reason for the existence of `window_`?

Reply:
Hmm ... maybe chat on this in Slack. Test seams do have some advantages over Sinon stubs so we may want to use different approaches in different scenarios.